public/app/features/alerting/unified/__snapshots__/PanelAlertTabContent.test.tsx.snap

```diff
@@ -13,10 +13,6 @@ exports[`PanelAlertTabContent Will render alerts belonging to panel and a button
     },
   ],
   "condition": "C",
-  "folder": {
```

Comment:
Probably because of the change we did in here: https://github.com/grafana/grafana/pull/68839/files#diff-3bbeeca14a940ec22888d7e5f44fd250b0b1670c494ca95506de27b4fe8d6cd4R27